packages/postcss-helpers-functions/package.json

```diff
@@ -0,0 +1,416 @@
+        "rollup-pluginutils": "^2.8.2",
+        "sade": "^1.8.1",
+        "safe-array-concat": "^1.1.3",
+        "safe-buffer": "^5.2.1",
+        "safe-identifier": "^0.4.2",
+        "safe-push-apply": "^1.0.0",
+        "safe-regex-test": "^1.1.0",
+        "semver": "^6.3.1",
+        "serialize-javascript": "^4.0.0",
+        "set-function-length": "^1.2.2",
+        "set-function-name": "^2.0.2",
+        "set-proto": "^1.0.0",
+        "shebang-command": "^1.2.0",
+        "shebang-regex": "^1.0.0",
+        "shell-quote": "^1.8.2",
+        "side-channel": "^1.1.0",
+        "side-channel-list": "^1.0.0",
+        "side-channel-map": "^1.0.1",
+        "side-channel-weakmap": "^1.0.2",
+        "signal-exit": "^4.1.0",
+        "slash": "^3.0.0",
+        "source-map": "^0.6.1",
+        "source-map-js": "^1.2.1",
+        "source-map-support": "^0.5.21",
+        "sourcemap-codec": "^1.4.8",
+        "spdx-correct": "^3.2.0",
+        "spdx-exceptions": "^2.5.0",
+        "spdx-expression-parse": "^3.0.1",
+        "spdx-license-ids": "^3.0.20",
+        "stable": "^0.1.8",
+        "string-hash": "^1.1.3",
+        "string-width": "^4.2.3",
+        "string-width-cjs": "^4.2.3",
+        "string.prototype.matchall": "^4.0.12",
+        "string.prototype.padend": "^3.1.6",
+        "string.prototype.trim": "^1.2.10",
+        "string.prototype.trimend": "^1.0.9",
+        "string.prototype.trimstart": "^1.0.8",
+        "strip-ansi": "^6.0.1",
+        "strip-ansi-cjs": "^6.0.1",
+        "strip-bom": "^3.0.0",
+        "style-inject": "^0.3.0",
+        "stylehacks": "^5.1.1",
+        "supports-color": "^7.2.0",
+        "supports-preserve-symlinks-flag": "^1.0.0",
+        "svgo": "^2.8.0",
+        "terser": "^5.37.0",
+        "tiny-glob": "^0.2.9",
+        "tslib": "^2.8.1",
+        "typed-array-buffer": "^1.0.3",
+        "typed-array-byte-length": "^1.0.3",
+        "typed-array-byte-offset": "^1.0.4",
+        "typed-array-length": "^1.0.7",
+        "typescript": "^4.9.5",
+        "unbox-primitive": "^1.1.0",
+        "undici-types": "^6.20.0",
+        "unicode-canonical-property-names-ecmascript": "^2.0.1",
+        "unicode-match-property-ecmascript": "^2.0.0",
+        "unicode-match-property-value-ecmascript": "^2.2.0",
+        "unicode-property-aliases-ecmascript": "^2.1.0",
+        "universalify": "^2.0.1",
+        "update-browserslist-db": "^1.1.1",
+        "util-deprecate": "^1.0.2",
+        "validate-npm-package-license": "^3.0.4",
+        "which": "^1.3.1",
+        "which-boxed-primitive": "^1.1.1",
+        "which-builtin-type": "^1.2.1",
+        "which-collection": "^1.0.2",
+        "which-typed-array": "^1.1.18",
+        "wrap-ansi": "^7.0.0",
+        "wrap-ansi-cjs": "^7.0.0",
+        "wrappy": "^1.0.2",
+        "y18n": "^5.0.8",
+        "yallist": "^3.1.1",
+        "yaml": "^1.10.2",
+        "yargs": "^17.7.2",
+        "yargs-parser": "^21.1.1"
```

Comment:
This is an absurd number of dependencies for a helper library.

There is no way these are all necessary.